pages/api/graphql-rest.page.ts

```diff
@@ -133,25 +129,6 @@ class MpdxRestApi extends RESTDataSource {
     return getTaskAnalytics(data);
   }
 
-  async getCoachingAnswerSets(
```

Comment:
I assume this came from a bad merge? Or your branch isn't up to date? I think this was merged to main today.

It's also why you've got TS errors.